Shadowlands/TazaveshTheVeiledMarket/Hylbrande.lua

```diff
@@ -5,39 +5,50 @@
 
 local mod, CL = BigWigs:NewBoss("Hylbrande", 2441, 2448)
 if not mod then return end
-mod:RegisterEnableMob(175663) -- Hylbrande
+mod:RegisterEnableMob(
+	175663, -- Hylbrande
+	175667  -- Titanic Defense Turret
+)
 mod:SetEncounterID(2426)
 mod:SetRespawnTime(30)
 
 --------------------------------------------------------------------------------
 -- Initialization
 --
 
+local sanitizingCycleTime = 0
+
 function mod:GetOptions()
 	return {
 		346971, -- [DNT] Summon Vault Defender
+		353312, -- Purifying Burst
 		{346116, "TANK"}, -- Shearing Swings
 		347094, -- Titanic Crash
-		346957, -- Purged by Fire
+		{346957, "SAY"}, -- Purged by Fire
 		346766, -- Sanitizing Cycle
 	}
 end
 
 function mod:OnBossEnable()
+	self:RegisterEvent("CHAT_MSG_RAID_BOSS_WHISPER")
```

Comment:
Do I need to unregister this anywhere?

Reply:
No, anything you register in OnEnable is unregistered when the boss is defeated.